CRM/Core/BAO/Email.php

```diff
@@ -334,4 +334,29 @@ public static function del($id) {
     return CRM_Contact_BAO_Contact::deleteObjectWithPrimary('Email', $id);
   }
 
+  /**
```

Comment:
Oh hang on, the field IS a boolean & there is setting which I never knew about & which is making me sad 'Enable multiple bulk email address for a contact'

```
  <field>
    <name>on_hold</name>
    <export>true</export>
    <type>boolean</type>
    <default>0</default>
    <required>true</required>
    <comment>Is this address on bounce hold?</comment>
    <add>1.1</add>
    <html>
      <type>CheckBox</type>
    </html>
  </field>
```